### ISSUE TITLE
Problem: am-devtools are not pakaged

### ISSUE THREAD

**Issue by scollazo** [artefactual-labs/am-packbuild#137]
As am-devtools are needed in updates, we should make available as deb/rpm package

**Comment by sevein** [comment]
Before we start working on this package can you please try to enumerate the tools that are needed to complete the AM17 upgrade? I feel like the tools in that repo fall in two categories:

1. Tools needed during the upgrade of Archivematica.
2. Maintenance tools needed by the application administrator.
2. Tools needed by developers.

In AM17 we were trying to convert the tools under the first category into formal management commands. Only rebuild-transfer-backlog came up and it's already added: https://github.com/artefactual/archivematica/tree/stable/1.7.x/src/dashboard/src/main/management/commands. If there's a tool missing we will convert them before the release. That will remove the need for this extra package.

Does that make sense?

**Comment by sevein** [comment]
The maintenance tools could be converted into Django commands too. This is where they're mentioned in the docs: https://www.archivematica.org/en/docs/archivematica-1.7/admin-manual/maintenance/maintenance/. But we could also just update the docs with notes on how to install devtools manually.

**Comment by sevein** [comment]
But if they're necessary during the upgrade like it seems to be based on what I'm seeing in this issue https://github.com/artefactual/archivematica-devtools/pull/50 I think then they're migration tools and they should fall into the first category and be converted into management commands.

**Comment by sevein** [comment]
Added `wontfix` label. Tentatively this is going to be fixed by converting the tools into management commands.
